lib/ansible/modules/cron.py

```diff
@@ -236,9 +237,9 @@ def __init__(self, module, user=None, cron_file=None):
 
         if cron_file:
             if os.path.isabs(cron_file):
-                self.cron_file = cron_file
+                self.cron_file = to_text(cron_file, errors='surrogate_or_strict')
```

Comment:
We should use bytes with file paths since the text type will vary between Python 2 (`unicode`) and Python 3 (`str`). We also prefix byte strings with `b_` as a convention.
```suggestion
                self.cron_file = cron_file
                self.b_cron_file = to_bytes(cron_file, errors='surrogate_or_strict')
```

This likely means there are other places that should use the bytes version of the path that need to be updated.